node_modules/gh-orgunit/tests/test-orgunit.js

```diff
@@ -523,7 +524,18 @@ describe('Organisational units', function() {
                         assert.strictEqual(part.canManage, true);
                         assert.strictEqual(module.canManage, true);
 
-                        return callback();
+                        // Verify the series have a `canManage` property
+                        OrgUnitTestUtil.assertGetOrgUnits(global.tests.admins.cam2013.client, null, true, partUnit.id, null, function(orgUnits) {
+                            assert.strictEqual(orgUnits.results.length, 1);
+                            assert.strictEqual(orgUnits.results[0].id, moduleUnit.id);
+
+                            assert.strictEqual(orgUnits.results[0].Series.length, series.length);
+                            _.each(orgUnits.results[0].Series, function(serie) {
```

Comment:
Maybe we can do an `assertSerie` function that also checks the `canManage` property?